src/main/webapp/WEB-INF/jsp/roster.jsp

```diff
@@ -122,26 +121,30 @@
                                 <input id="saveAttendanceOnTop" class="hovering-purple-button" type="submit" name="saveAttendance"
                                        value="Save Attendance"/>
                             </div>
-                            <div class="col-md-4 deleteAttendanceButton">
+                            <div class="col-md-3 deleteAttendanceButton">
                                 <a id="deleteAttendance" href="${context}/roster/${selectedSectionId}/delete" name="deleteAttendance" style="text-decoration: none" >
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this deprecated "name" attribute. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3AS1827)